recipes/aioboto3/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "aioboto3" %}
+{% set version = "9.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: abac5dcfa871627b7040e6586a69e3359e2dfc4e15dc66135969f2d26fbdcb3b
+
+build:
+  number: 0
+  noarch: python
+  script: rm -f poetry.lock && {{ PYTHON }} -m pip install --no-deps .
+
+requirements:
+  build:
+    - python >=3.7,<3.9
+    - pip
+    - poetry >=1.1.11,<2.0.0a
+    - poetry-dynamic-versioning >=0.13.1,<0.14.0a
+  run:
+    - python >=3.7
+    - aiobotocore >=2.3.0
+    - cryptography >=2.3.1
+    - chalice >=1.24.0
+    - boto3 >=1.24.21
+    - s3transfer >=0.4.0,<0.7.0a
+    - idna >=2.10
```

Comment:
```suggestion
  run:
    # base runtime environment
    - python >=3.7
    - pip

    # boto direct functional dependencies
    - aiobotocore >=2.3.0            # Async core layer for AWS python bindings
    - boto3 >=1.20.0                    # AWS API bindings for Python
    - s3transfer >=0.5.0,<0.6.0a  # python module to move files to and from AWS S3

    # General web service infrastructure modules
    - idna >=3.3                           # International Domain Names - for URL host formatting
    - cryptography >=37.0.0        # Python module that provide bindings for data encryption primatives
    - chalice >=1.27.0                 # serverless app framework - client side for comms to AWS API

```

Reply:
It's a little unusual to document these yaml so heavily.